src/core/Rule.ml

```diff
@@ -67,7 +67,7 @@ end = struct
   let to_string x = x
 
   let validate =
-    let rex = SPcre.regexp "^[a-zA-Z0-9._-]*$" in
+    let rex = SPcre.regexp "^[a-zA-Z0-9._-@]*$" in
```

Comment:
Probably need to have the `-` last otherwise we specify a range of chars.

```suggestion
    let rex = SPcre.regexp "^[a-zA-Z0-9._@-]*$" in
```

Reply:
Ah right! Great call.